File: tests/06_phases_openmp.dg/nanox/c/success_target_01.c
/*
<testinfo>
test_generator=config/mercurium-nanox
</testinfo>
*/

#define N 10

void hi (int i);

int main (int argc, char *argv[])
{
  int i;

  for (i = 0; i < N; i++) {
#pragma omp target device(smp_numa) copy_deps
#pragma omp task
    hi(i);
  }
}

File: tests/01_fortran.dg/success_target_01.f90
! <testinfo>
! test_generator=config/mercurium-fortran
! </testinfo>
PROGRAM MAIN
    IMPLICIT NONE
    INTEGER  :: X, Y, Z
    TARGET :: X(10)
    TARGET :: Y(10), Z(20)

    X(1) = 2
    Y(3) = 4
    Z(5) = 6
END PROGRAM MAIN
